specification/communication/data-plane/CallAutomation/stable/2023-03-06/communicationservicescallautomation.json

```diff
@@ -1500,6 +1341,9 @@
     },
     "GetParticipantsResponse": {
       "description": "The response payload for getting participants of the call.",
+      "required": [
+        "values"
+      ],
       "type": "object",
       "properties": {
         "values": {
```

Comment:
The property should be called `value` singular, not `values`.
https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#collections